akita/src/plugins/paginator/paginator-plugin.ts

```diff
@@ -287,9 +287,15 @@ export class PaginatorPlugin<E> extends AkitaPlugin<E> {
 
         const { range, pagesControls } = this.config;
 
-        if (!this.pagination.total) {
-          response.total = response.perPage * response.lastPage;
-          this.pagination.total = response.total;
+        this.pagination.total = response.total;
```

Comment:
I'm not sure what you did here. You need to call it once after you set the `response.total`.

Reply:
sry for this misunderstanding